### PR TITLE
blog: Cody VS Code v0.12 

### PR DESCRIPTION
**Pull request by kalanchan**
blog for vs code v0.12

### CODE REVIEW

File: content/blogposts/2023/cody-vscode-0-12-release.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/cody-vscode-0.12.0-og-image.jpg
+socialImage: https://storage.googleapis.com/sourcegraph-assets/cody-vscode-0.12.0-og-image.jpg
+---
+
+[Cody for VS Code](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai) v0.12 is now available and includes a new onboarding & sign in experience, more custom configuration options, and improved Cody performance.
+
+### User Onboarding and Sign-in Experience
+
+New users can now sign up for Cody using their Github and Gitlab account.
```
Comment (abeatrix) [inline content/blogposts/2023/cody-vscode-0-12-release.md:23]:
```suggestion
New users can now sign up and sign in for Cody using their Github and Gitlab accounts. 
```

Comment (kalanchan) [reply content/blogposts/2023/cody-vscode-0-12-release.md]: ty!

File: content/blogposts/2023/cody-vscode-0-12-release.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+[Cody for VS Code](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai) v0.12 is now available and includes a new onboarding & sign in experience, more custom configuration options, and improved Cody performance.
+
+### User Onboarding and Sign-in Experience
+
+New users can now sign up for Cody using their Github and Gitlab account. 
+
+### Cody Pre-Instructions
+
+You can now add set a custom starter message that prepend chat requests to Cody by setting the `cody.chat.preInstruction` configuration in `settings.json`.
```
Comment (abeatrix) [inline content/blogposts/2023/cody-vscode-0-12-release.md:27]:
```suggestion
You can now add a custom starter message that prepend chat requests to Cody by setting the `cody.chat.preInstruction` configuration in `settings.json`.
```

**Review by abeatrix — APPROVED** [review]

File: content/blogposts/2023/cody-vscode-0-12-release.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+New users can now sign up and sign in for Cody using their Github and Gitlab accounts. 
+
+### Cody Pre-Instructions
+
+You can now add a custom starter message that prepend chat requests to Cody by setting the `cody.chat.preInstruction` configuration in `settings.json`.
+
+```json
+{
+    "cody.chat.preInstruction": "My name is Jeff."
```
Comment (abeatrix) [inline content/blogposts/2023/cody-vscode-0-12-release.md:31]: maybe a better example lol?

Comment (abeatrix) [reply content/blogposts/2023/cody-vscode-0-12-release.md:31]:
```suggestion
    "cody.chat.preInstruction": "I am a frontend engineer."
```

Comment (kalanchan) [reply content/blogposts/2023/cody-vscode-0-12-release.md:31]: i took it from you! 😆 

let me add to it

**Pull request merged**